style.css

```diff
@@ -406,6 +407,129 @@ footer {
   width: 30px;
 }
 
+.get-button1 {
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 12px;
+  color: #fff;
+  background-color: #36b37e;
+  text-align: center;
+  margin: auto;
+  padding: 20px;
+  border-radius: 4px;
+  display: flex;
+  width: 140px;
+  justify-content: center;
+  border: none;
+}
+
+.get-button1:hover {
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+}
+
+.get-button1:active {
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+  background: #008552;
+}
+
+.flex {
+  display: flex;
+}
+
+.error-message {
+  color: #f00;
+}
+
+#carddetailsmain {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  z-index: 1000;
+  background-color: rgba(128, 128, 128, 0.5);
+  display: none;
+}
+
+#carddetailsmain p {
+  width: 100%;
+  text-align: left;
+}
+
+#carddetails {
+  background-color: #fff;
+  width: 690px;
+  left: 0;
+  top: 0;
+  border-radius: 8px;
+  padding: 20px;
+}
+
+.capture {
+  width: 100%;
+  border-radius: 8px;
+  height: 300px;
+  object-fit: cover;
+  border: none;
+}
+
+.dclosebtn {
+  margin: 0 0 8px auto;
+  display: flex;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
```

Comment:
- [ ] Some portions of the popup window are not showing on the top and bottom sides on the mobile screen sizes. Let's ensure to improve upon this aspects accordingly.

**Your design:**        |  **The original design:**
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/12745474/68dfc664-be69-4ea1-948d-14d658acf3f8"> | <img width="800" alt="Home Page" src="https://github.com/Wils-FOURR-speed/My-portfolio/assets/12745474/72bd8915-c76d-4144-b68a-2ebc77980ff8">